Client/src/Components/NavigationMenu.tsx

```diff
@@ -55,7 +55,7 @@ export default function NavigationMenu() {
     return user && user.firstName ?
       (
         <Typography>
-          Hello, {user.firstName} {user.lastName}
+          {user.firstName} {user.lastName}&nbsp;&nbsp;&nbsp;
```

Comment:
remove the special symbols, do it with the help of a grid instead, check how grid is done in DefaultFormFooter for example

Reply:
@ds161833  done